embassy-usb/src/control.rs

```diff
@@ -106,7 +106,7 @@ impl Request {
         let recipient = rt & 0b11111;
 
         Request {
-            direction: if rt & 0x80 == 0 { Direction::Out } else { Direction::In },
+            direction: rt.into(),
```

Comment:
I think it makes sense to keep the conversions local, but I have a separate commit for this in case the previous is preferred.